### PR TITLE
[READY]Sends the names of online admins when sending IRC alerts about no admins active.

### PR DESCRIPTION
**Pull request by silicons**
ADMIN: Owegno: So I can yell at people AFK
Oh wait that'd include me but i'm doing it anyways because i'm bored.
WIP: making it work in other places that use the same proc.

### CODE REVIEW

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -132,27 +132,27 @@
...
-	. = adm["present"]
+	. = adm["present"].len
 	if(. <= 0)
 		var/final = ""
-		if(!adm["afk"] && !adm["stealth"] && !adm["noflags"])
+		if(!adm["afk"].len && !adm["stealth"].len && !adm["noflags"].len)
 			final = "[msg] - No admins online"
 		else
-			final = "[msg] - All admins AFK ([adm["afk"]]/[adm["total"]]), stealthminned ([adm["stealth"]]/[adm["total"]]), or lack[rights2text(requiredflags, " ")] ([adm["noflags"]]/[adm["total"]])"
+			final = "[msg] - All admins stealthed\[[english_list(adm["stealth"])]\], AFK\[[english_list(adm["afk"])]\], or lacks +BAN\[english_list(adm["noflags"])]\]! Total: adm["present"].len "
```
Comment (Cyberboss) [inline code/modules/admin/verbs/adminhelp.dm]: You're missing some escapes methinks

**Comment by kevinz000** [comment]
ok yeah il fix this soon.

**Comment by PKPenguin321** [comment]
Now change it so we can kick AFK trialadmins

**Comment by FantasticFwoosh** [comment]
Nowhere to hide now.

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -215,7 +215,8 @@ var/datum/subsystem/ticker/ticker
 			qdel(S)
 
 	var/list/adm = get_admin_counts()
-	if(!adm["present"])
+	var/list/allmins = adm["present"]
+	if(!allmins.len)
```
Comment (optimumtact) [inline code/controllers/subsystem/ticker.dm]: why is this changed?

Comment (kevinz000) [inline code/controllers/subsystem/ticker.dm]: Doesn't work otherwise.
Will throw errors if I just use adm["present"].len

Comment (optimumtact) [inline code/controllers/subsystem/ticker.dm]: oh because you changed it to an empty list by default?
